xblock/test/test_parsing.py

```diff
@@ -123,24 +129,24 @@ class ParsingTest(XmlTest, unittest.TestCase):
 
     @XBlock.register_temp_plugin(Leaf)
     def test_parsing(self):
-        block = self.parse_xml_to_block("<leaf data2='parsed'/>")
+        block = self.parse_xml_to_block(u"<leaf data2='parsed'/>")
```

Comment:
Yep.  I thought I had cleaned it all up.  I'll make another pass to see what else I missed.